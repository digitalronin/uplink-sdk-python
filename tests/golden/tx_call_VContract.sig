r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 36328708371944199253089606500124086907610551451514837967349346827373936077775
