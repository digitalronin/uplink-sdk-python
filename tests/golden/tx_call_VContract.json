{
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
    "header": {
        "tag": "TxContract", 
        "contents": {
            "tag": "Call", 
            "contents": {
                "args": [
                    {
                        "tag": "VContract", 
                        "contents": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve"
                    }
                ], 
                "method": "get", 
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve"
            }
        }
    }, 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE0zNjMyODcwODM3MTk0NDE5OTI1MzA4OTYwNjUwMDEyNDA4NjkwNzYxMDU1MTQ1MTUxNDgzNzk2NzM0OTM0NjgyNzM3MzkzNjA3Nzc3NQ=="
}